lib/Utilities/utils.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+
+class Utils {
+  static final messengerKey = GlobalKey<ScaffoldMessengerState>();
```

Comment:
U PR za autentikaciju sam ovo spomenuo, koristite `ScaffoldMessenger.of(context)` umjesto keya